static/generate_i18n_resources_list.py

```diff
@@ -39,7 +40,11 @@ def get_translation_info(filepath):
             continue
         print("Processing", i18n_file.name)
         if i18n_file.name != "test.json":
-            language_list.append(get_translation_info(i18n_file))
+            lang_code, lang_name = get_translation_info(i18n_file)
+            if lang_name is not None:
```

Comment:
An empty string `lang_name` is also a invalid value.
Better using
```python
if lang_name:
    language_list.append(...)
....
```

Reply:
Fixed